src/AdminConsole/Pages/Organization/Log.cshtml

```diff
@@ -6,38 +6,104 @@
 }
 
 <div>
-    @if (Model.Events.Any())
-    {
-        <table class="shadow-sm bg-white text-center">
-            <thead class="border text-sm">
-            <tr>
-                <th>Performed By</th>
-                <th>Performed At</th>
-                <th>Severity</th>
-                <th>Message</th>
-                <th>Subject</th>
-                <th>Event Type</th>
-            </tr>
-            </thead>
-            <tbody class="border text-sm">
-            @foreach (var auditEvent in Model.Events)
-            {
-                <tr class="border">
-                    <td>@auditEvent.PerformedBy</td>
-                    <td>@auditEvent.PerformedAt.ToLocalTime()</td>
-                    <td>@auditEvent.Severity</td>
-                    <td>@auditEvent.Message</td>
-                    <td>@auditEvent.Subject</td>
-                    <td>@auditEvent.EventType</td>
-                </tr>
-            }
-            </tbody>
-        </table>
-    }
-    else
-    {
-        <p>This organization has no events from the past @Model.RetentionPeriod days.</p>
-    }
+    <div class="flow-root" asp-if="Model.Events.Any()">

```

Comment:
Just curious: Is there a benefit to using `asp-if` over `@if (...)`?

Also should the paging section at the bottom be changed to `asp-if` to be consistent?

Reply:
@jrmccannon The only benefit to using `asp-if` is to reduce nesting and visual noise. Yeah, I think paging could be changed as well.